### PR TITLE
[FindPw] Improve the coverage of `FindPw` page

### PR DESCRIPTION
**Pull request by geoseong**
## Specify project
`react-native`

## Description

Add test case of covering mutation's `onError` function

referred to [this "relay" documentation](https://relay.dev/docs/guides/testing-relay-components/#example-with-queueoperationresolver)

## Related Issues

none

### Cases
- Should show error alert when user does not exists
- Should show error alert when email sent failed

### Coverage
**[Before]**
image

**[After]**
image

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/hackatalk/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] Run `yarn lint && yarn tsc`
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: client/src/components/pages/__tests__/FindPw.test.tsx [hunk 1]
```diff
@@ -57,6 +57,54 @@ describe('[FindPw] interaction', () => {
       expect(errorText).toBeTruthy();
     });
 
+    it('should show error alert when user does not exists', async () => {
+      const mockEnvironment = createMockEnvironment();
+
+      mockEnvironment.mock.queueOperationResolver(
+        () => new Error(JSON.stringify({message: 'user does not exists'})),
+      );
```
Comment (0916dhkim) [inline client/src/components/pages/__tests__/FindPw.test.tsx:65]: I did not know this is how you mock an error in Relay. #TIL

**Review by 0916dhkim — APPROVED** [review]
Looks good to me. We need more contributions that increase the test coverage 😄

**Comment by geoseong** [comment]
and I've found extra information about it.

The error message shouldn't be wrapped in JSON.

that test code can be passed too.

**before**
```
  mockEnvironment.mock.queueOperationResolver(
    () => new Error(JSON.stringify({message: 'user does not exists'})),
  );
```
**after**
```
  mockEnvironment.mock.queueOperationResolver(
    () => new Error('user does not exists'),
  );
```

**Review by hyochan — APPROVED** [review]
LGTM!

**Pull request merged**